packages/rx_bloc_generator/pubspec.yaml

```diff
@@ -1,35 +1,35 @@
 name: rx_bloc_generator
 description: Code generator for rx_bloc that makes your BloC zero-boilerplate.
-version: 4.0.0
+version: 5.0.0
 repository: https://github.com/Prime-Holding/rx_bloc/tree/master/packages/rx_bloc_generator
 issue_tracker: https://github.com/Prime-Holding/rx_bloc/issues
 homepage: https://www.primeholding.com/
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
@DDavidPrime, do we need to constraint the devs to use 2.13.0, or rx_bloc will be running fine with 2.12 as well?